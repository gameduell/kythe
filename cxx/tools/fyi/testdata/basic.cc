#include <stdio.h>

int main() {
  foo();
  return 0;
}
