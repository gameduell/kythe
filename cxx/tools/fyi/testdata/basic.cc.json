{"source":{"path":"basic.h"},"edge_kind":"","fact_name":"/kythe/node/kind","fact_value":"L2t5dGhlL25vZGUvZmlsZQ=="}
{"source":{"signature":"SOMENODE"},"edge_kind":"","fact_name":"/kythe/node/kind","fact_value":"L2t5dGhlL25vZGUvZnVuY3Rpb24="}
{"source":{"signature":"SOMEANCHOR"},"edge_kind":"","fact_name":"/kythe/node/kind","fact_value":"L2t5dGhlL25vZGUvYW5jaG9y"}
{"source":{"signature":"foo#n"},"edge_kind":"","fact_name":"/kythe/node/kind","fact_value":"L2t5dGhlL25vZGUvbmFtZQ=="}
{"source":{"signature":"SOMENODE"},"edge_kind":"/kythe/edge/named","target":{"signature":"foo#n"},"fact_name":"/","fact_value":""}
{"source":{"signature":"SOMEANCHOR"},"edge_kind":"/kythe/edge/defines","target":{"signature":"SOMENODE"},"fact_name":"/","fact_value":""}
{"source":{"signature":"SOMEANCHOR"},"edge_kind":"/kythe/edge/childof","target":{"path":"basic.h"},"fact_name":"/","fact_value":""}
