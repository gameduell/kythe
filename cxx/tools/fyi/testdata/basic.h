#ifndef BASIC_H
#define BASIC_H

void foo();

#endif
