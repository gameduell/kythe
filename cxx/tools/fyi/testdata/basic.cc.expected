#include <stdio.h>
#include "basic.h"


int main() {
  foo();
  return 0;
}
